### ISSUE TITLE
Investigate Multi-Target Regression Models

### ISSUE THREAD

**Issue by sam-mata** [sam-mata/evolution-antarctica#77]
### Details
While unlikely to yield any performance benefits, it may be worthwhile to cover multi-target approaches for contrast as a baseline to the sequential models produced in #60 .

**Comment by sam-mata** [comment]
# Results
## Ice Thickness
| Model | MSE | RMSE | MAE | R2 |
|-------|-----|------|-----|----|
| Multi-target ExtraTrees | 0.000642 | 0.025343 | 0.008879 | 0.990057 |
| Multi-target RandomForest | 0.000756 | 0.027490 | 0.008547 | 0.988302 |
| Multi-target Linear Regression | 0.007701 | 0.087756 | 0.060153 | 0.880783 |
| Multi-target Ridge | 0.007701 | 0.087755 | 0.060152 | 0.880783 |
| Multi-target XGBoost | 0.011933 | 0.109238 | 0.091619 | 0.815272 |
| Multi-target LightGBM | 0.012103 | 0.110014 | 0.092334 | 0.812638 |
| Multi-target GradientBoosting | 0.013929 | 0.118020 | 0.096844 | 0.784374 |
| Multi-target ElasticNet | 0.053446 | 0.231185 | 0.197646 | 0.172615 |
| Multi-target Lasso | 0.064599 | 0.254163 | 0.217919 | -0.000034 |
| Multi-target MLPRegressor | 0.289873 | 0.538398 | 0.410515 | -3.487417 |

## Ice Velocity
| Model | MSE | RMSE | MAE | R2 |
|-------|-----|------|-----|----|
| Multi-target ExtraTrees | 0.000076 | 0.008741 | 0.001116 | 0.528462 |
| Multi-target RandomForest | 0.000091 | 0.009520 | 0.001172 | 0.440695 |
| Multi-target LightGBM | 0.000119 | 0.010890 | 0.002545 | 0.268137 |
| Multi-target GradientBoosting | 0.000145 | 0.012038 | 0.002800 | 0.105600 |
| Multi-target Linear Regression | 0.000157 | 0.012547 | 0.003300 | 0.028451 |
| Multi-target Ridge | 0.000157 | 0.012547 | 0.003300 | 0.028451 |
| Multi-target Lasso | 0.000162 | 0.012730 | 0.003320 | -0.000190 |
| Multi-target ElasticNet | 0.000162 | 0.012730 | 0.003320 | -0.000190 |
| Multi-target XGBoost | 0.000169 | 0.012999 | 0.002879 | -0.042789 |
| Multi-target MLPRegressor | 0.123782 | 0.351827 | 0.308238 | -762.928056 |

## Ice Mask
| Model | MSE | RMSE | MAE | R2 |
|-------|-----|------|-----|----|
| Multi-target ExtraTrees | 0.003535 | 0.059453 | 0.014034 | 0.984569 |
| Multi-target RandomForest | 0.003935 | 0.062727 | 0.013074 | 0.982823 |
| Multi-target LightGBM | 0.040738 | 0.201835 | 0.187511 | 0.822156 |
| Multi-target XGBoost | 0.041059 | 0.202630 | 0.188063 | 0.820753 |
| Multi-target Linear Regression | 0.042598 | 0.206393 | 0.150208 | 0.814034 |
| Multi-target Ridge | 0.042598 | 0.206393 | 0.150207 | 0.814034 |
| Multi-target GradientBoosting | 0.045505 | 0.213320 | 0.195905 | 0.801342 |
| Multi-target ElasticNet | 0.111886 | 0.334493 | 0.311609 | 0.511552 |
| Multi-target MLPRegressor | 0.110393 | 0.332254 | 0.271859 | 0.518070 |
| Multi-target Lasso | 0.191213 | 0.437279 | 0.421482 | 0.165243 |
